games-arcade/jumpnbump/jumpnbump-1.61.recipe

```diff
@@ -0,0 +1,69 @@
+SUMMARY="Play cute bunnies jumping on each other's heads"
+DESCRIPTION="This is a game for the whole family. You play cute fluffy little \
+bunnies and hop on each other's heads.
+
+At the beginning you are in the menu, where you have to let each active \
+player juimp over the tree trunk to enter the play area, and then walk to the \
+right. You will then enter the arena. The aim is to jump on the other bunnies' \
+heads...
+
+Jump 'n Bump was originally a DOS game by Brainchild Design, which was open \
+sourced under the GPL license and ported to SDL, and then SDL 2."
+HOMEPAGE="https://gitlab.com/LibreGames/jumpnbump"
+COPYRIGHT="1998-2019 Jump 'n Bump developers"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="$HOMEPAGE/-/archive/1.61/jumpnbump-1.61.zip"
+CHECKSUM_SHA256="0bd5dbd22f81b9399c6e6dbc372c950e09ee19fe348d5f54f5abe8d4794f6490"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+SECONDARY_ARCHITECTURES="x86"
+commandBinDir=$prefix/bin
+fi
+
+PROVIDES="
+	jumpnbump$secondaryArchSuffix = $portVersion
+	cmd:gobpack
+	cmd:jnbpack
+	cmd:jnbunpack
+	cmd:jumpnbump
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libbz2$secondaryArchSuffix
+	lib:libSDL2_2.0$secondaryArchSuffix
+	lib:libSDL2_net_2.0$secondaryArchSuffix
+	lib:libSDL2_mixer_2.0$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libbz2$secondaryArchSuffix
+	devel:libSDL2$secondaryArchSuffix
+	devel:libSDL2_net$secondaryArchSuffix
+	devel:libSDL2_mixer$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	"
+
+BUILD()
+{
+	make $jobArgs \
+		PREFIX="$prefix" \
+		BINDIR="$commandBinDir" \
+		DATADIR="$dataDir"
+}
+
+INSTALL()
+{
+	make PREFIX="$prefix" BINDIR="$commandBinDir" DATADIR="$dataDir" install
```

Comment:
Maybe?
```diff
+	rm -rf $dataDir/{applications,icons}
```

Reply:
Yeah, I wasn't sure about that. Done (didn't use brace expansion as it's a Bashism and fbrosson wanted me to avoid them when they were here).

Reply:
Was the first solution that came to mind ;)